<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dimension</class>
 <widget class="QWidget" name="Dimension">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>401</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hbox">
    <item>
     <widget class="QPushButton" name="playButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="posButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>t = 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="slider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="pageStep">
       <number>50</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mergeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Merge</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fpsButton">
      <property name="text">
       <string>10 fps</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
